src/utils/issueRedeem.ts

```diff
@@ -185,21 +201,22 @@ export async function redeem(
     btcRelayAPI: BTCRelayAPI,
     redeemingAccount: KeyringPair,
     amount: MonetaryAmount<WrappedCurrency, BitcoinUnit>,
-    vaultAddress?: string,
+    nativeCurrency: CollateralCurrency,
+    vaultId?: InterbtcPrimitivesVaultId,
     autoExecute = ExecuteRedeem.Auto,
     network: BitcoinNetwork = "regtest",
     atomic = true,
     timeout = 5 * 60 * 1000
 ): Promise<Redeem> {
     const bitcoinjsNetwork = getBitcoinNetwork(network);
-    const redeemAPI = new DefaultRedeemAPI(api, bitcoinjsNetwork, electrsAPI, amount.currency);
+    const redeemAPI = new DefaultRedeemAPI(api, bitcoinjsNetwork, electrsAPI, amount.currency, nativeCurrency);
     redeemAPI.setAccount(redeemingAccount);
 
     const btcAddress = "bcrt1qujs29q4gkyn2uj6y570xl460p4y43ruayxu8ry";
     const [redeemRequest] = await redeemAPI.request(
         amount,
         btcAddress,
-        vaultAddress ? newAccountId(api, vaultAddress) : undefined,
+        vaultId,
         atomic,
         0 // retries
```

Comment:
Minor, but if you need to comment this to describe what it is then it might be cleaner to declare it as a variable, i.e. `const retries = 0`.